### DISCUSSION TITLE
$say helper

### DISCUSSION THREAD

**Discussion by SimoTod** [alpine-collective/alpine-magic-helpers#49] [Ideas] [answered]
Are we interested in something like https://codepen.io/SimoTod/pen/LYRPber?
It's probably something people won't use much but it's cool 😂 
P.s. use chrome for now

**Comment by KevinBatdorf** [comment, marked as answer]
You have to change `dix` to `div` in your demo.

It's kind of cool. Maybe think about it a bit. Maybe also `$speak()` instead? Sounds more retro.

On the same note, I think sounds effects might start being more common over the next decade. I wonder if there's something we could do to make that more convenient too.

**Reply by SimoTod** [reply]
Yeah, the underlying API uses "speak" but it sounds really formal (almost odd) to me. "please speak something" vs  "please say something". To be fair, I can't think of a real use case right now. Maybe I'll create an Alpine-audio-helpers thingy just for fun but I'll keep it separated for now

**Reply by KevinBatdorf** [reply]
> Maybe I'll create an Alpine-audio-helpers thingy just for fun but I'll keep it separated for now

I think that's fine, and I think in the future we could have 100+ helpers included once we get all the super useful ones out (which act like marketing too). Then we can build some documentation to order them in a smart way, and have hundreds, and maybe a core, essentials, and other bundle
